test/jdk/java/foreign/StdLibTest.java

```diff
@@ -58,6 +66,13 @@ public class StdLibTest extends NativeTestHelper {
 
     private StdLibHelper stdLibHelper = new StdLibHelper();
 
+    final static Charset nativeCharset = Charset.forName(System.getProperty("native.encoding"));
+
+    static MemorySegment nativeBytesFromString(Arena arena, String str) {
+        byte[] nativeBytes = str.getBytes(nativeCharset);
+        return arena.allocateArray(ValueLayout.JAVA_BYTE, Arrays.copyOf(nativeBytes, nativeBytes.length + 1));
+    }
```

Comment:
Please move this before line 67 to keep all static features together.

Reply:
Moved this in https://github.com/ibmruntimes/openj9-openjdk-jdk21/compare/960e06a3679336e039a146ae1ebe4fcfa56da55c..a837b75d57e86bb113c40ee75d4738a6af6e1e8d